rbs/sa.tf

```diff
@@ -0,0 +1,51 @@
+# The service account for the cloudsql proxy.
+resource "google_service_account" "sqlproxy" {
+  count = var.enable ? 1 : 0
+
+  provider     = google.target
+  project      = var.google_project
+  account_id   = "${local.service}-${local.owner}-sqlproxy"
+  display_name = "${local.service}-${local.owner}-sqlproxy"
+}
+resource "google_project_iam_member" "sqlproxy_role" {
+  count = var.enable ? 1 : 0
+
+  provider = google.target
+  project  = var.google_project
+  role     = "roles/cloudsql.client"
+  member   = "serviceAccount:${google_service_account.sqlproxy[0].email}"
+}
+
+locals {
+  app_sa_roles = [
+    # Allow creating project.
+    "roles/resourcemanager.projectCreator",
```

Comment:
projectCreator is not a google_project_iam_member role, it belongs on a folder or organization.
We're also gonna need more roles to set up billing accounts, and probably other things like buckets, but maybe best to start with the simplest pieces first, like you have.

Reply:
Thanks for pointing the folder permission. For now, I simply put a folder role and make folder_id as input variables. 
In the future, we may want to come back and update here to support multiple folders for different clients. 
And for now to support RBS integration test, I will also enable the service for toolsalpha and create a rbs-testing folder under it.